tf-job-operator-chart/Chart.yaml

```diff
@@ -1,8 +1,7 @@
 name: tf-job-operator-chart
 home: https://github.com/jlewi/mlkube.io
-version: 0.1.0
-appVersion: 0.1.0
-description: K8s Third Party Resource and Operator For TensorFlow jobs
-icon: https://raw.githubusercontent.com/hashicorp/consul/bce3809dfca37b883828c3715b84143dd71c0f85/website/source/assets/images/favicons/android-chrome-512x512.png
+version: v1alpha1
```

Comment:
Will this break?
> non-SemVer names are explicitly disallowed by the system.
 https://github.com/kubernetes/helm/blob/master/docs/charts.md#charts-and-versioning

Also I think it would make sense to use a separate version number for the chart to help debug future issues.
For example a future chart might include the dashboard at some point. If users are experiencing issues, asking for the installed chart version will help us debug.

Reply:
Good catch. I changed it looks like names like `version: 1.2.3-alpha.1+ef365` are supported. I think it will be useful to include the sha at which the release was built; e.g

```
tf-job-operator-chart-0.1.0-v20171020-a1d0405.tgz 
```